packages/vue/src/components/atoms/SfBreadcrumbs/SfBreadcrumbs.vue

```diff
@@ -10,17 +10,22 @@
         <template v-if="last !== i">
           <!-- @slot Custom markup for previous pages (binds `breadcrumb` object) -->
           <slot name="link" v-bind="{ breadcrumb, go }">
-            <a class="sf-breadcrumbs__breadcrumb" @click="go(breadcrumb)">{{
-              breadcrumb.text
-            }}</a>
+            <SfLink
+              :link="breadcrumb.route.link"
```

Comment:
@justyna-13 we can use here `breadcrumb.link`, what did you think?

Reply:
sure :)